bundles/org.openhab.binding.adorne/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.adorne-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
```

Comment:
Done

Reply:
Actually, after I made this change I started getting the following build error. Do you know what that means?

[INFO] --- karaf-maven-plugin:4.2.7:verify (karaf-feature-verification) @ org.openhab.binding.adorne ---
[INFO] Using repositories: https://openhab.jfrog.io/openhab/libs-release@id=openhab-release,https://openhab.jfrog.io/openhab/libs-snapshot@id=openhab-snapshot@noreleases@snapshots,https://repo.maven.apache.org/maven2@id=central
[WARNING] Error resolving artifact org.openhab.core.features.karaf:org.openhab.core.features.karaf.openhab-core:jar:3.0.0-SNAPSHOT: [Could not find artifact org.openhab.core.features.karaf:org.openhab.core.features.karaf.openhab-core:jar:3.0.0-SNAPSHOT in openhab-snapshot (https://openhab.jfrog.io/openhab/libs-snapshot/)]
java.io.IOException: Error resolving artifact org.openhab.core.features.karaf:org.openhab.core.features.karaf.openhab-core:jar:3.0.0-SNAPSHOT: [Could not find artifact org.openhab.core.features.karaf:org.openhab.core.features.karaf.openhab-core:jar:3.0.0-SNAPSHOT in openhab-snapshot (https://openhab.jfrog.io/openhab/libs-snapshot/)]

Reply:
The problem is the wrong base branch. Your PR is against master (which is on 3.0.0). You need to rebase on `2.5.x`. If unsure how to do that, please ask, otherwise chances are good you mess up the whole branch. I can assist if needed.

Reply:
Worked fine as you said after I rebased. Done.